restapi/pom.xml

```diff
@@ -158,11 +158,6 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-        <groupId>org.mockito</groupId>
-        <artifactId>mockito-junit-jupiter</artifactId>
-        <scope>test</scope>
-    </dependency>
```

Comment:
same here? I am pretty sure we use classes from this package in the tests?

Reply:
Removal of duplicate entry